cg/server/admin.py

```diff
@@ -441,6 +442,49 @@ class AnalysisView(BaseView):
     form_extra_fields = {"workflow": SelectEnumField(enum_class=Workflow)}
 
 
+class IlluminaFlowCellView(BaseView):
+    """Admin view for Model.IlluminaSequencingRun"""
+
+    column_list = (
```

Comment:
I think we should extend these to also show the aggregated metrics i.e. reads/ yield for the run so production can easily see get these

Reply:
add all columns